### ISSUE TITLE
Need written introduction to FOSC

### ISSUE THREAD

**Issue by anubhavp28** [IIITKalyaniFOSC/iiitkalyanifosc.github.io#5]
We need help in coming up with a kickass written introductory content to Free and Open Source Community. It should IMO focus on atleast -

* Motivation behind FSOC
* Goals (deadset things we want to do in next academic year)
* Little history how it started (maybe?)

We can use this comment section (on this issue) to take part in a discussion. Suggestions on additional points to include in introductory content  are, for sure welcome.

**Comment by anubhavp28** [comment]
Please ask for suggestions here, before submitting a PR?

**Comment by zeus0789** [comment]
History (source: Wikipedia)
In 1997, Eric Raymond published The Cathedral and the Bazaar, a reflective analysis of the hacker community and free software principles. The paper received significant attention in early 1998, and was one factor in motivating Netscape Communications Corporation to release their popular Netscape Communicator Internet suite as free software. This code is today better known as Mozilla Firefox and Thunderbird.

Netscape's act prompted Raymond and others to look into how to bring the FSF's free software ideas and perceived benefits to the commercial software industry. 
The Open Source Initiative was founded in February 1998 to encourage use of the new term and evangelize open-source principles.[28]

While the Open Source Initiative sought to encourage the use of the new term and evangelize the principles it adhered to, commercial software vendors found themselves increasingly threatened by the concept of freely distributed software and universal access to an application's source code






Welcome to the Free and open source community.
A community where we feel rewarded by sharing and evolving Technology. A community where not a single line of code is charged. The sole purpose of such a community is to create software for which the source code can be freely shared, with anyone, for any purpose.
Free and open source softwares matters,  especially for students and developers because it's real and because the same can look into the heart of the code. Thus he can learn from it and customise it according to his personal needs.
Without FOSS, anyone looking for a real world experience in contributing to the world of open source software development will face disappointment. Thus, FOSS provides decreased software cost and above all free education and a chance to get involved with the software community.
With this in mind, we at the Indian Institute of Information Technology Kalyani, laid foundation for the free and open source community of the college known as FOSC( free and open source community) , with the goal to further extend what some of the greatest minds in the history of programming started. We aim to work as a community and provide our juniors (and anyone interested) the chance to get involved with some real world development right from the freshman term. 




This is a preview. If you want I can work on this and submit a pr

**Comment by zeus0789** [comment]
Text from" Welcome to the free...." Is written by me

**Comment by PulkitMishra** [comment]
@zeus0789 check this out https://www.reddit.com/r/explainlikeimfive/comments/21ik6j/eli5_what_is_open_source_software_and_why_is_it/

**Comment by zeus0789** [comment]
it's great @PulkitMishra , probably the best explanation i've ever read.

**Comment by PulkitMishra** [comment]
Try paraphrasing it.
